### DISCUSSION TITLE
Compatibility with last libreLink

### DISCUSSION THREAD

**Discussion by pkviet** [NightscoutFoundation/xDrip#3069] [Q&A]
Hello
I'm using xdrip+ with Freestyle libre 2 EU.
But I also need to keep the capability to use LibreLink since my diabetologist at hospital relies on the reports from LibreLink.
I've always taken care to disable LibreLink for BT to be connected to xdrip+ which updates readings every minute as a cgm.
However, Abbott has updated LibreLink so that through BT it's now capable to act as cgm and display readings every minute.
My question is whether both xdrip+ and LibreLink will be able to be used at the same time or if a single one can connect to BT and act as cgm.
If anyone has feedback, i'd be interested since I'm scheduled to change my patch in a few days.

**Comment by pkviet** [comment]
I'll give the answer to my own question.
After applying a new Freestyle libre 2 EU patch, I found out that I could still use both xdrip+ and LibreLink. Readings every minute work on both of them so the patch BT pairing is not limited to a single app. I made sure though to follow the previous manner of linking xdrip+ to the patch by (1) starting the patch with LibreLink, (2) force stop LibreLink, (3) nfc scan after one hour in xdrip+ and then hit start in xdrip+, which after 3 readings leads to the calibration screen. (4) reopen LibreLink.

**Reply by KeijoMika** [reply]
Did you have "Nearby devices" permission enabled for LibreLink all the time? I am about to start my first sensor with the updated app and also wondering if xDrip will still collect data. As am using it for Smartwatch functionality.

**Reply by pkviet** [reply]
> Did you have "Nearby devices" permission enabled for LibreLink all the time? I am about to start my first sensor with the updated app and also wondering if xDrip will still collect data. As am using it for Smartwatch functionality.

Yeah i had nearby devices enabled all the time for LibreLink iirc. But i had force stopped it for one hour after starting the patch with it.
I do use xdrip+ with watch integration (pixel watch). Works great. Everything is updated every minute

**Reply by SmallStrayFox** [reply]
hey there @pkviet 

i'm just looking into using xdrip+ so i can set up nightscout, but like you i'm concerned about being able to continue to use LibreLink. Did you use any particular guide to set up xdrip+ to work alongside LibreLink? is it still working as it should be 7+ months on?

**Reply by pkviet** [reply]
> hey there @pkviet 
> 
> i'm just looking into using xdrip+ so i can set up nightscout, but like you i'm concerned about being able to continue to use LibreLink. Did you use any particular guide to set up xdrip+ to work alongside LibreLink? is it still working as it should be 7+ months on?

Yeah, it's working fine. I just followed the recipe outlined in previous posts. I'm on alpha channel though.
